src/components/PlaceholderContainer/__stories__/PlaceholderContainer.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import type {Meta, StoryFn} from '@storybook/react';
+
+import {PlaceholderContainer} from '../PlaceholderContainer';
+
+import {PlaceholderContainerShowcase} from './PlaceholderContainerShowcase';
+
+export default {
+    title: 'Components/Data Display/PlaceholderContainer',
+    component: PlaceholderContainer,
+    parameters: {
+        a11y: {
+            element: '#storybook-root',
+            config: {
+                rules: [
+                    {
+                        id: 'color-contrast',
+                        enabled: false,
+                    },
+                ],
+            },
+        },
+    },
+} as Meta;
+
+const ShowcaseTemplate: StoryFn = () => <PlaceholderContainerShowcase />;
+export const Showcase = ShowcaseTemplate.bind({});
```

Comment:
Let's have separate stories for each prop of the component (size, direction, image, action) like we do in most other component's stories